.docker/plugins/cinder/config.json

```diff
@@ -0,0 +1,136 @@
+{
+      "Args": {
+        "Description": "",
+        "Name": "",
+        "Settable": null,
+        "Value": null
+      },
+      "Description": "REX-Ray for Cinder Persistent Disks",
+      "Documentation": "https://github.com/codedellemc/rexray/.docker/plugin/cinder",
```

Comment:
Hi @sebglon,

Oops, it looks like this is an issue across all of the plug-ins. Please correct yours, and I will correct the others.